examples/hello_openassetio.ipynb

```diff
@@ -0,0 +1,374 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "07297a3d-048b-496b-adb0-8fdd67316021",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Hosts: Hello OpenAssetIO\n",
+    "\n",
+    "This notebook illustrates how to initialise an OpenAssetIO session, resulting in a `Manager` instance for communication with a particular asset management system's plugin. We then perform a simple query of the manager, to `resolve` a property of an entity.\n"
```

Comment:
I think it is, in that the reader can play around with the notebook locally, if they wish.  So it's not like a traditional document. Ideally (and we've vaguely discussed in the past) we would put these notebooks on an interactive site, rather than just relying on the static GitHub rendered output.